apollo-compiler/src/test/graphql/com/example/directive_with_inline_fragment/TestQuery.java

```diff
@@ -340,7 +340,7 @@ public Hero read(ResponseReader reader) {
         )),
         ResponseField.forFragment("__typename", "__typename", Arrays.<ResponseField.Condition>asList(
           ResponseField.Condition.booleanCondition("withDetails", false),
-          ResponseField.Condition.typeCondition(new String[] {"Human", "Droid"})
+          ResponseField.Condition.typeCondition(new String[] {"Droid", "Human"})
```

Comment:
The order change is unfortunate but I don't think it matters a lot. I think it is due to moving to SDL and the order types are parsed. Where would be the good place to enforce iteration order? When parsing the schemas? Generating code? Somewhere else?

Reply:
It really doesn't matter in the behaviour. We could sort them alphabetically to maintain the order but not sure if it worth doing

Reply:
Ok let's keep this for now.